src/mod/features/repel_fix.cpp

```diff
@@ -0,0 +1,56 @@
+#include "exlaunch.hpp"
+
+#include "externals/Dpr/EvScript/EvDataManager.h"
+#include "externals/ItemWork.h"
+#include "externals/PlayerWork.h"
+
+#include "logger/logger.h"
+
+const int32_t REPEL_ITEM_ID = 79;
+const int32_t SUPER_REPEL_ITEM_ID = 76;
+const int32_t MAX_REPEL_ITEM_ID = 77;
+
+const int32_t INFINITE_REPEL_FLAG = 2195;
+
+// Dpr.EvScript.EvDataManager$$
+HOOK_DEFINE_INLINE(RepelInventoryOverride){
+    static void Callback(exl::hook::nx64::InlineCtx* ctx)
+    {
+        auto evDataManager = (Dpr::EvScript::EvDataManager::Object*)ctx->X[0];
+        if (evDataManager->fields._eventListIndex == 0xFFFFFFFF)
+        {
+            auto label = (System::String::Object*)ctx->X[1];
+            auto callback = (Dpr::EvScript::EvDataManager::EventEndDelegate::Object*)ctx->X[2];
+            evDataManager->JumpLabel(label, callback);
+        }
+    }
+};
+
+HOOK_DEFINE_REPLACE(EncDataSave_CanUseSpray){
+    static bool Callback(MethodInfo *method)
```

Comment:
You can remove MethodInfo parameters for methods in 99% of cases. The exceptions will use a different more specific type like MethodInfo_1a2b3c.

You can remove them from your replace hook and your trampoline hook here.